docs/fundamentals/flow/yaml-spec.md

```diff
@@ -0,0 +1,66 @@
+(flow-yaml-spec)=
+# YAML specification
+
+This page outlines the specification for valid Flow YAML files.
+
+Such YAML configurations can be used to generate a Flow object via `Flow.load_config('flow.yml')`.
+
+To generate a YAML configuration from a `Flow` Python object, run `f.save_config()`.
+
+## Example
+
+The following constitutes an example Flow configuration:
+
+`flow.yml`/`flow.yaml`:
+```yaml
+jtype: Flow
+version: '1'
+with:
+  protocol: http
+metas:
+  - py_modules:
+      - executor.py
```

Comment:
This patterns does not seem to work. But also not sure if we even want to promote it.

Reply:
then remove if it does not seem to work